docs/community/contribute/nearcore.md

```diff
@@ -166,32 +167,69 @@ sys     0m4.307s
 
 Works only with nightly compiler.
 
-```bash
-# cargo clean
-# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y' RUSTC_WRAPPER= cargo build -p neard --release
+```console
+$ cargo clean
+$ time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y' RUSTC_WRAPPER= cargo build -p neard --release
 real    1m42.999s
 user    33m31.341s
 sys     1m25.773s
-# touch */*/*/*.rs
-# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y' RUSTC_WRAPPER= cargo build -p neard --release
+$ touch */*/*/*.rs
+$ time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y' RUSTC_WRAPPER= cargo build -p neard --release
 real    0m23.501s
 user    4m30.597s
 sys     0m3.804s
 ```
 
+### Use system-provided rocksdb
+
+By default, building `neard` will result in compiling `librocksdb`
+which takes up some non-negligible amount of memory especially when
+building on a system with many cores.  It’s possible to configure
+build process to provide system-provided library instead.  This is
+done by setting `ROCKSDB_LIB_DIR` environment variable to location
+where `librocksdb.a` file is installed:
+
+```console
+$ cargo clean
+$ time ROCKSDB_LIB_DIR=/usr/lib/x86_64-linux-gnu cargo build -p neard --release
```

Comment:
nit: using the `env` command (`env VAR=val VAR2=val2 command...`) makes supplying these environment variables in this style less dependent on the specific shell used.